<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>743</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(204, 213, 174);
selection-background-color: rgb(169, 176, 143);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>841</width>
      <height>691</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget::pane {
	boder: 1px;
	background: rgb(204, 213, 174);
}

QTabBar::tab {
	background: rgb(227, 231, 196);
	min-width: 200 px;
	min-height: 30 px;
	margin-left: 1px;
	left: -1px;
}

QTabBar::tab:selected {
	background: rgb(204, 213, 174);
}

QTabBar::tab:hover {
	background: rgb(233, 237, 201);
}

</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="edit_tab">
     <property name="font">
      <font>
       <family>Century Gothic</family>
      </font>
     </property>
     <attribute name="title">
      <string>Редактировать мероприятия</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>831</width>
        <height>581</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(204, 213, 174);
QHeaderView
{
	background-color: rgb(169, 176, 143);
}

QHeaderView::section
{
	background-color: rgb(169, 176, 143);
	font: 9pt &quot;Century Gothic&quot;;
}

QTableWidget::item::selected 
{
 
	color: rgb(254, 253, 239);
	background-color: rgb(105, 109, 88);
	font: 8pt &quot;Century Gothic&quot;;
}
 </string>
      </property>
     </widget>
     <widget class="QPushButton" name="changeButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>597</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(105, 109, 88);
color: rgb(255, 255, 255);
font: 10pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Изменить</string>
      </property>
     </widget>
     <widget class="QLabel" name="syntax_label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>610</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="add_tab">
     <attribute name="title">
      <string>Создать мероприятие</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>0</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Описание мероприятия:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>490</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Афиша:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Место проведения:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_placeButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>211</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(105, 109, 88);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Добавить место проведения</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>320</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Дата и время проведения:</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="description">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>30</y>
        <width>431</width>
        <height>481</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(234, 241, 210);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="createButton">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>590</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(105, 109, 88);
color: rgb(255, 255, 255);
font: 10pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string>Создать</string>
      </property>
     </widget>
     <widget class="QPushButton" name="imgButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>510</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(105, 109, 88);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Загрузить изображение</string>
      </property>
     </widget>
     <widget class="QComboBox" name="typeBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(234, 241, 210);</string>
      </property>
      <item>
       <property name="text">
        <string>Все</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Тип мероприятия:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="placeBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(234, 241, 210);</string>
      </property>
      <item>
       <property name="text">
        <string>Все</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="add_typeButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(105, 109, 88);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Создать новый тип</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>390</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>с:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>430</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>до:</string>
      </property>
     </widget>
     <widget class="QTimeEdit" name="begintimeEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>380</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(234, 241, 210);
font: 9pt &quot;Century Gothic&quot;;</string>
      </property>
     </widget>
     <widget class="QTimeEdit" name="endtimeEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>420</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(234, 241, 210);
font: 9pt &quot;Century Gothic&quot;;</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>340</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(234, 241, 210);
font: 9pt &quot;Century Gothic&quot;;</string>
      </property>
     </widget>
     <widget class="QLabel" name="error_label">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>550</y>
        <width>271</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;Century Gothic&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Century Gothic</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Название:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="name">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>31</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(234, 241, 210);
font: 9pt &quot;Century Gothic&quot;;
border: 1px solid rgb(105, 109, 88);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="picture1">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>530</y>
        <width>201</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="picture2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>560</y>
        <width>161</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>836</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
